hatch/api_client.py

```diff
@@ -23,6 +23,21 @@ def create_release(workspace, release, user):
     return proxy_httpx_response(response)
 
 
+def upload_file(release_id, name, path, user):
```

Comment:
Could you explain what and why this endpoint is doing what it's doing in a docstring?  I remember agreeing with you this made sense before but have entirely forgotten why it needed to do this, sorry!

Reply:
Have added